less/api.less

```diff
@@ -123,32 +116,33 @@ table.arguments {
     padding-right: 0;
   }
 
-  tr.argument--depth-1 {
```

Comment:
This was arguably not BEM-like originally; the proper BEM class name would probably be `arguments__argument--depth-1`, or perhaps `arguments__row--depth-1` if we wanted to be less verbose. I'd be cool with it if you wanted to do something like that.

Reply:
I've gone with your second suggestion here in my most recent commits